product_warranty/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Warranty",
+    "summary": "Allow user to specify a warranty on a product",
+    "version": "11.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "category": "Product",
+    "website": "http://www.opensourceintegrators.com",
+    "depends": ["product", "stock"],
```

Comment:
Why do you put stock if there is no reference to it anywhere?

Reply:
copy&paste error :P